src/promptflow/promptflow/_core/run_tracker.py

```diff
@@ -171,10 +171,26 @@ def _flow_run_postprocess(self, run_info: FlowRunInfo, output, ex: Optional[Exce
 
     def _update_flow_run_info_with_node_runs(self, run_info):
         run_id = run_info.run_id
-        run_info.api_calls = self._collect_traces_from_nodes(run_id)
         child_run_infos = self.collect_child_node_runs(run_id)
         run_info.system_metrics = run_info.system_metrics or {}
         run_info.system_metrics.update(self.collect_metrics(child_run_infos, self.OPENAI_AGGREGATE_METRICS))
+        # TODO: Refactor Tracer to support flow level tracing,
+        # then we can remove the hard-coded root level api_calls here.
+        # It has to be a list for UI backward compatibility.
+        start_timestamp = run_info.start_time.astimezone(timezone.utc).timestamp() \
+            if run_info.start_time else None
+        end_timestamp = run_info.end_time.astimezone(timezone.utc).timestamp() \
+            if run_info.end_time else None
+        run_info.api_calls = [{
+            "name": "flow_root",
+            "node_name": "flow_root",
```

Comment:
* Add `"type": "Flow"`? (Newly define an entry in the `TraceType` enum)
* Do we need to add `"error"` if the flow run failed? (We can do this later if it is not required for now)
Below is a sample trace created with Tracer, just for your reference:
![image](https://github.com/microsoft/promptflow/assets/113209/cdf53168-c11d-4931-941c-ed94500acbd5)

Reply:
Good point. Added Type and error.